packages/api/src/store/stream-table.ts

```diff
@@ -296,7 +296,17 @@ export default class StreamTable extends Table<DBStream> {
 
   async getLastSession(id: string, opts?: QueryOptions): Promise<DBStream> {
     const res: QueryResult<DBLegacyObject> = await this.db.queryWithOpts(
-      sql`SELECT data FROM stream  WHERE data->>'parentId'=${id} ORDER BY data->'createdAt' DESC LIMIT 1`.setName(
+      sql`SELECT data FROM stream  WHERE data->>'parentId'=${id} ORDER BY data->>'createdAt' DESC LIMIT 1`.setName(
+        `${this.name}_by_parentid_last_session`
+      ),
+      opts
+    );
+    return res.rowCount < 1 ? null : (res.rows[0].data as DBStream);
+  }
+
+  async getLastSessionFromSessionsTable(id: string, opts?: QueryOptions) {
```

Comment:
This function should be in a `sessions-table.ts` file/class instead, while this class accesses the `stream` table. The whole parent/child stream/session is already pretty confusing, we should not make them look even more interchangeable.